src/sessions/screen-recording-session-handler.ts

```diff
@@ -73,15 +73,37 @@ export default class ScreenRecordingSessionHandler extends BaseSessionHandler {
   }
 
   private updateScreenRecordingMetadatas (session: ScreenRecordingMediaSession, metadatas: ScreenRecordingMetadata[]) {
+    metadatas.forEach((meta) => {
+      // adding any here because I don't want to add this to the public interface
+      (meta as any)._trackId = meta.trackId;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9596369)
```suggestion
      (meta as unknown)._trackId = meta.trackId;
```